### ISSUE TITLE
EFFECT_STATIC and REPEATING types not working

### ISSUE THREAD

**Issue by SuperiorHome0** [GC-spigot/AdvancedEnchantments#3802]
### Describe the bug

EFFECT_STATIC and REPEATING both do not work with
these effects I was testing:

- POTION:<POTION>:<LEVEL>:[TICKS]
- POTION_OVERRIDE:<POTION>:<LEVEL>:[TICKS]
- LAVA_WALKER
- WATER_WALKER
- FLY

When the type is used there are no error commands in console
or any feedback from the plugin in-game, and the enchantments do not work.

### How to reproduce

To reproduce I pasted in the glowing enchantment and it simply did not work at all

glowing:
  display: '%group-color%Glowing'
  description: Gives permanent night vision.
  applies-to: Helmet
  type: EFFECT_STATIC
  group: SIMPLE
  applies:
    - ALL_HELMET
  levels:
    '1':
      effects:
        - POTION:NIGHT_VISION:0
  
Down below is the original enchantment I tested, and both the default enchantment glowing and the one I created produce the same results.

### Screenshots / Videos

![image](https://github.com/GC-spigot/AdvancedEnchantments/assets/150862319/0fdf74ec-50df-4e66-bf62-26c7ab05eb6c)

### "/ae plinfo" link

https://paste.md-5.net/jolesiguva

### Server Log

https://mclo.gs/Uprv2Fv

**Comment by austv** [comment]
@SuperiorHome0 latest version is working fine, try this instead:

```
  type: 'EFFECT_STATIC'
  group: 'CELESTIALBLESSINGS'
  applies:
    - ALL_CHESTPLATE
  levels:
    1:
      effects:
        - 'POTION:FAST_DIGGING:0'
```

**Comment by SuperiorHome0** [comment]
Oh that actually works pretty well thank you, One more thing I think I realized why they weren't working. I did some testing and it didn't seem to be the actual enchantment but an error with how the plugin reloads? I'm not quite sure how it works.

Here's what I found though, apparently when I add a new enchantment with the repeating or effect static type then reload the plugin simply just makes the enchantment not work at all. 

![image](https://github.com/GC-spigot/AdvancedEnchantments/assets/150862319/c06708a1-dd65-40db-86ab-0741abdefc6e)

^ I added the enchantment above to my enchantments.yml and for some reason it didn't work at all after reloading, only after restarting my server did it start sending an error message that the enchantment itself had syntax errors

**Comment by SuperiorHome0** [comment]
I also found that changing the types **sometimes** breaks the enchantment and other times it does not, even when using the proper syntax for the type change.

The only solution I see for now is restarting my server everytime I make any change or add a new effect_static / repeating type enchant just for it to work as intended, which is very very not ideal lol

**Comment by ThomasWega** [comment]
The EXP effect was broken in that version of AE. Try with the latest. If there is any issue with the actual reloading please create a new issue and I will handle it straight away with a high priority
